### PR TITLE
cronet: update README for using published grpc-cronet with play service Cronet implementation

### PR DESCRIPTION
**Pull request by voidzcy**

### CODE REVIEW

**Comment by ejona86** [comment]
Make sure to backport to release branch.

File: cronet/README.md [hunk 1]
```diff
@@ -17,13 +16,26 @@ Some advantages of using Cronet with gRPC:
...
+Google Play Services Client Library for Cronet
+
+```
+implementation 'io.grpc:grpc-cronet:1.24.0'
+implementation 'com.google.android.gms:play-services-cronet:16.0.0'
+```
+
+In cases where Cronet cannot be loaded from Google Play services, there is a less performant 
+implementation of Cronet's API that can be used. Depend on `org.chromium.net:cronet-fallback` 
+to use this fall-back implementation,
```
Comment (dapengzhang0) [inline cronet/README.md]: Period?

File: cronet/README.md [hunk 2]
```diff
@@ -17,13 +16,26 @@ Some advantages of using Cronet with gRPC:
...
+See the example app at https://github.com/GoogleChrome/cronet-sample/blob/master/README.md.
 
-To use Cronet, you must have the `ACCESS_NETWORK_STATE` permission set in
+## Example usage:
+
+In your app module's `build.gradle` file, include a dependency on both `grpc-cronet` and the 
+Google Play Services Client Library for Cronet
+
+```
+implementation 'io.grpc:grpc-cronet:1.24.0'
```
Comment (dapengzhang0) [inline cronet/README.md]: How do we maintain the version in the doc for new releases? Add 
```
$ ${EDITOR:-nano -w} cronet/README.md
```

to RELEASING.md?

Comment (voidzcy) [reply cronet/README.md]: Seems we do not sync the version number in README.md. 

I saw grpc-android's doc is also outdated. We may want to add that to RELEASING.md.

Comment (dapengzhang0) [reply cronet/README.md]: Which file is grpc-android's doc you mentioned?

Comment (voidzcy) [reply cronet/README.md]: https://github.com/grpc/grpc-java/blob/master/documentation/android-channel-builder.md#example-usage

Comment (dapengzhang0) [reply cronet/README.md]: That is up to date. You only need to bump its version during releasing process.

Comment (voidzcy) [reply cronet/README.md]: Oh, you are right. That's done in tagging the release. We could do the same for cronet by adding 
```
$ ${EDITOR:-nano -w} cronet/README.md
```

File: cronet/README.md (hunk 1 above)
Comment (voidzcy) [reply cronet/README.md]: It's a period. Changed. Thanks for catching it.

File: cronet/README.md (hunk 2 above)
Comment (voidzcy) [reply cronet/README.md]: Added the version bump command to RELEASING.md.

**Pull request merged**